### PR TITLE
Update for SR2022 Competition

### PR DESCRIPTION
**Pull request by Scarzy**
Includes some re-organising of the structure

### CODE REVIEW

File: competition/setup.tex [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+	available.\\
+}}
+{\makecell{
+Health and Safety Lead\\
+SR Volunteers \\
+}}
+{3}
+\hline
+
+\risk{Injury from improper manual handling}
```
Comment (RealOrangeOne) [inline competition/setup.tex:99]: > manual handling

What other kind of manual handling is there?

**Comment by RealOrangeOne** [comment]
Looks like the renames have broken CI

**Comment by Scarzy** [comment]
> Looks like the renames have broken CI

Fixed

File: competition/setup.tex (hunk 1 above)
Comment (Scarzy) [reply competition/setup.tex:99]: proper manual handling..?
I'm not sure I see what you're asking?

**Review by RealOrangeOne — APPROVED** [review]

**Review by sedders123 — APPROVED** [review]
All looks good and changes in de8a45e match what we discussed on Slack and in the Meet call

**Pull request merged**